kratos/processes/find_nodal_neighbours_process.h

```diff
@@ -79,11 +74,20 @@ class FindNodalNeighboursProcess
     /// avg_elems ------ expected number of neighbour elements per node.,
     /// avg_nodes ------ expected number of neighbour Nodes
     /// the better the guess for the quantities above the less memory occupied and the fastest the algorithm
```

Comment:
I would update the comment. Basically remove it!

Reply:
ok